src/wp-admin/includes/post.php

```diff
@@ -778,16 +778,18 @@ function get_default_post_to_edit( $post_type = 'post', $create_in_db = false )
  * @param string $date    Optional. Post date.
  * @param string $type    Optional. Post type.
  * @param string $status  Optional. Post status.
+ * @param integer $author Optional. Post author ID.
  * @return int Post ID if post exists, 0 otherwise.
  */
-function post_exists( $title, $content = '', $date = '', $type = '', $status = '' ) {
+function post_exists( $title, $content = '', $date = '', $type = '', $status = '', $author = '' ) {
 	global $wpdb;
 
 	$post_title   = wp_unslash( sanitize_post_field( 'post_title', $title, 0, 'db' ) );
 	$post_content = wp_unslash( sanitize_post_field( 'post_content', $content, 0, 'db' ) );
 	$post_date    = wp_unslash( sanitize_post_field( 'post_date', $date, 0, 'db' ) );
 	$post_type    = wp_unslash( sanitize_post_field( 'post_type', $type, 0, 'db' ) );
 	$post_status  = wp_unslash( sanitize_post_field( 'post_status', $status, 0, 'db' ) );
+	$post_author = wp_unslash( sanitize_post_field( 'post_author', $author, 0, 'db' ) );
```

Comment:
```suggestion
	$post_author  = wp_unslash( sanitize_post_field( 'post_author', $author, 0, 'db' ) );
```

WordPress Coding Standards want us to line up the equals signs here.